components/collapse/Collapse.tsx

```diff
@@ -9,8 +9,8 @@ import animation from '../_util/openAnimation';
 export type ExpandIconPosition = 'left' | 'right';
 
 export interface CollapseProps {
-  activeKey?: Array<string> | string;
-  defaultActiveKey?: Array<string>;
+  activeKey?: Array<string> | string | number;
+  defaultActiveKey?: Array<string | number>;
```

Comment:
@thylsky Should be:
```
activeKey?: Array<string | number> | string | number;
defaultActiveKey?: Array<string | number> | string | number;
```

Reply:
Yes, it was kinda confusing to me, why the defaultActiveKey expects only array.

Reply:
But the documentation says that it doesn't expects an array...

Reply:
@thylsky activeKey supports single value and array value both and `defaultActiveKey`'s type is as same as `activeKey`.

Reply:
@dengfuping Fixed. 👍I've also updated the documentation related to this.